pkgs/development/libraries/qt-5/5.7/srcs.nix

```diff
@@ -122,6 +122,14 @@
       name = "qtimageformats-opensource-src-5.7.1.tar.xz";
     };
   };
+  qtinstaller = {
```

Comment:
It looks like qtinstaller is not part of the Qt library scenarios, but as it is also delivered by upstream and behaves like the other submodules, I would keep it inside the package of qt libraries.